wp-content/themes/core/components/blocks/gallery_slider/css/gallery_slider.pcss

```diff
@@ -0,0 +1,110 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Component: Gallery Slider
+ *
+ * ----------------------------------------------------------------------------- */
+
+.b-gallery-slider {
+	padding-bottom: var(--spacer-70); /* should blocks have vertical padding? */
```

Comment:
There are some spacing vars defined for this; but tbh, the whole system needs a spacing pass. you can leave this as-is.